network/containerizer/bridgepolicy.go

```diff
@@ -380,11 +405,13 @@ func (p *BridgePolicy) PopulateContainerLinkLayerDevices(m Machine, containerMac
 			}
 		}
 	}
-	missingSpace := containerSpaces.Difference(spacesFound)
+
+	guestSpaceSet := set.NewStrings(guestSpaces.Names()...)
+	missingSpace := guestSpaceSet.Difference(spacesFound)
 
 	// Check if we are missing "" and can fill it in with a local bridge
 	if len(missingSpace) == 1 && missingSpace.Contains("") && p.containerNetworkingMethod == "local" {
```

Comment:
Perhaps the `""` could be replaced by `network.DefaultSpaceName` to be more consistent?

Reply:
Done.